polynote-frontend/polynote/ui.js

```diff
@@ -574,6 +574,52 @@ export class NotebookCellsUI extends UIEventTarget {
     }
 }
 
+class EditBuffer {
+
+    constructor() {
+        this.versions = [];
+    }
+
+    /**
+     * Add edits corresponding to a version. The version should always increase.
```

Comment:
> The version should always increase.

Do we want to enforce this here?

Reply:
Will address that during further work on the versioning stuff